izumi-reflect/izumi-reflect/src/main/scala_3/izumi/reflect/TagMacro.scala

```diff
@@ -83,11 +80,11 @@ class TagMacro(using val ctx: Quotes) {
       case _ => List(tpe)
     }
 
-  private def summonTag(typeRepr: TypeRepr): Expr[Tag[?]] =
+  private def summonTag[any <: AnyKind](typeRepr: TypeRepr): Expr[Tag[any]] =
     typeRepr.asType match {
-      case '[a] =>
```

Comment:
@kitlangton 
`case '[a]` produced a MatchError here when trying to compile `simple combined Tag` test. Seems like quoted match expects `a` to be a proper type and fails. `case given Type[a]` doesn't fail however